pandas/core/interchange/from_dataframe.py

```diff
@@ -513,5 +514,10 @@ def set_nulls(
             # cast the `data` to nullable float dtype.
             data = data.astype(float)
             data[null_pos] = None
+        except SettingWithCopyError:
+            # SettingWithCopyError happens if the `data` appears
+            # to have 'NaT'. If this happens, copy the `data`.
```

Comment:
"appears to" sounds a bit mysterious :) how about
```
`SettingWithCopyError` may happen for datetime-like with missing values
```

Reply:
thank you, I changed the wording as you suggested.